css/styles.css

```diff
@@ -31,6 +138,50 @@ ul {
   margin-bottom: 0;
 }
 
+.contact {
+  background-image: url(../images/Contact.png);
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  text-align: center;
+}
```

Comment:
- [ ] [Required] Please Add proper margin to the footer (Contact) Section as shown in the Figma design. This makes your portfolio looks more beautiful.